maintenance.py

```diff
@@ -81,30 +119,33 @@ def wait_for_maintenance(callback):
             last_maintenance_event = maintenance_event
             callback(maintenance_event)
 
-
 def maintenance_callback(event):
     if event:
-        # print('Undergoing host maintenance: {}'.format(event))
+        logger.warning('Undergoing host maintenance: %s', event)
         # realistically, any sort of maintenence event should drain aerospike
         asinfo1 = [ASINFO, "-v", "quiesce:"]
         asinfo1.extend(args.options.split())
```

Comment:
Can you also rename `asinfo1` and `asinfo2` to be just `asinfo`.

```
    if event:
        logger.warning('Undergoing host maintenance: %s', event)
        # realistically, any sort of maintenence event should drain aerospike
        asinfo = [ASINFO, "-v", "quiesce:"]
        asinfo.extend(args.options.split())
        run_shell_command(asinfo)
        logger.info('quiesce finished')

    else:
        logger.info('Finished host maintenance')
        asinfo = [ASINFO, "-v", "quiesce-undo:"]
        asinfo.extend(args.options.split())
        run_shell_command(asinfo)
        logger.info('quiesce-undo finished')
```

Reply:
i know, it's just to make sure we don't step on the variable (we had such a bug in the code before - with the asinfo and the asadm the comes later). is it important to reuse the name?